### ISSUE TITLE
Improve checking Content-Type header value

### ISSUE THREAD

**Issue by Leo506** [ArbusBiz/Arbus.Network#42]
Now if you want to check what content-type header value response has you need to write something like this:
```c#
if (response.Content.Headers.ContentType.MediaType == HttpContentType.Application.Json)
{
    // Do something
}
```
It looks too wordy. It would be cool if you can write something like this:
```c#
if (response.HasApplicationJsonContentType())
{
    // Do something
}
```

So, I suggest to add extension methods for checking content-type header value to `HttpResponseMessage `

**Comment by evgenyvalavin** [comment]
I agree there's a way to make checking for content type more readable.

First, I suggest we check HttpResponseMessage.Content's content type because HttpReponseMessage doesn't have a content type itself.
Second, HttpResponseMessage.Content.IsJson() method needs to be as much simple as it can be.
Our job is not to introduce new inaccuracies, but rather to make it simple and clear.

**Comment by Leo506** [comment]
Ok, let's create extension methods for `HttpContent` class where we wiil check content type and name its like `Has<Content Type>ContentType` (e.g. HasApplicationJsonContentType(), HasTextPlainContentType()). I think it is the most clear name that could be

**Comment by evgenyvalavin** [comment]
I think HasTextPlainContentType or HasApplicationJsonContentType methods are long.

HttpResponseMessage.Content.HasTextPlainContentType() - it's obvious that we check content not any other properties, hence "ContentType" in method name is redundant.

"Has" doesn't sound natural for me, "Is" is more common and shorter.
"Has" sounds like Content has many content types and we check if in a collection of content types there's Json (for example) content type, whilst "Is" doesn't sound like there's a collection of content types.

**Comment by Leo506** [comment]
You convinced me.
Chaged PR
